packages/paste-icons/tools/templates/jsonTemplate.ts

```diff
@@ -1,10 +1,26 @@
+import {DEPRECATED_ICONS} from '../constants';
+
+type IconFiles = {
+  name: string;
+  svg: string;
+  cjs: string;
+  esm: string;
+};
+
+const createIconsFilesObject = (icons: string[] | undefined): IconFiles[] | undefined =>
+  icons?.map((iconName: string) => ({
+    name: iconName,
+    svg: `/svg/${iconName}.svg`,
+    cjs: `/cjs/${iconName}.js`,
+    esm: `/esm/${iconName}.js`,
+  }));
+
 export const jsonTemplate = (iconList: string[] | undefined): string => {
```

Comment:
Question: should this function be unit tested?

Reply:
Maybe. Its output is largely covered by the existing test, but if you think we could benefit from one I'd be happy to add it